### PR TITLE
Start menu ui

### PR DESCRIPTION
**Pull request by chrilehner**
changed layout from 1 column for predefined and custom named sets to 2 columns

### CODE REVIEW

File: src/styles/_start_menu.scss [hunk 1]
```diff
@@ -304,6 +295,24 @@
...
+        justify-content: space-between;
+
+        form {
+          width: 90%;
+        }
+
+        // fix select2 placeholder cutoff
+        // https://github.com/select2/select2/issues/291
+        .select2-search__field {
+          width: auto !important;
```
Comment (sgratzl) [inline src/styles/_start_menu.scss]: aren't you enforing a width of 100% when defining the search field?

File: src/FormBuilder.ts [hunk 2]
```diff
@@ -205,6 +205,11 @@ interface IFormElement extends IEventHandler {
   value: any;
 
   /**
+   * Form element values for multiple selection
+   */
+  values?: {id: string, text: string}[];
```
Comment (sgratzl) [inline src/FormBuilder.ts]: I would prefer moving this function as part of the `ISelectFormElement`

File: src/styles/_start_menu.scss (hunk 1 above)
Comment (chrilehner) [inline src/styles/_start_menu.scss]: yes I am, but the width of the textbox of select2 in which the user actually types somehow gets a width of 100px, which doesn't fit the placeholders

File: src/FormBuilder.ts (hunk 2 above)
Comment (chrilehner) [inline src/FormBuilder.ts]: `IFormSelectElement` currently needs methods that FormSelect2 does not implement (e.g. updateOptionElements), which may not be needed
Maybe a new `IFormSelect2Element` with the `values` method would be better?

**Pull request merged**